contrib/run_task.sh

```diff
@@ -205,45 +269,54 @@ build_docs_with_stable_toolchain() {
     RUSTDOCFLAGS="-D warnings" $cargo doc --all-features
 }
 
+# Bench only works with a non-stable toolchain (nightly, beta).
+do_bench() {
+    for crate in bitcoin hashes; do
+        pushd "$REPO_DIR/$crate" > /dev/null
+        RUSTFLAGS='--cfg=bench' cargo bench
+        popd > /dev/null
+    done
+}
+
+# This is only relevant for hashes.
+do_schemars() {
+    pushd "$REPO_DIR/hashes/extended_tests/schemars" > /dev/null
+    cargo test
+    popd > /dev/null
+}
+
 do_wasm() {
+    pushd "$REPO_DIR/hashes" > /dev/null
+
     clang --version &&
-	CARGO_TARGET_DIR=wasm cargo install --force wasm-pack &&
-	printf '\n[target.wasm32-unknown-unknown.dev-dependencies]\nwasm-bindgen-test = "0.3"\n' >> Cargo.toml &&
-	printf '\n[lib]\ncrate-type = ["cdylib", "rlib"]\n' >> Cargo.toml &&
-	CC=clang-9 wasm-pack build &&
-	CC=clang-9 wasm-pack test --node;
+        CARGO_TARGET_DIR=wasm cargo install --force wasm-pack &&
+        printf '\n' >> Cargo.toml &&
+        printf '[target.wasm32-unknown-unknown.dev-dependencies]\n' >> Cargo.toml &&
+        printf 'wasm-bindgen-test = "0.3.41"\n' >> Cargo.toml &&
+        CC=clang-9 wasm-pack build &&
+        CC=clang-9 wasm-pack test --node;
+
+    popd > /dev/null
 }
 
 do_asan() {
     cargo clean
     CC='clang -fsanitize=address -fno-omit-frame-pointer'                                        \
       RUSTFLAGS='-Zsanitizer=address -Clinker=clang -Cforce-frame-pointers=yes'                    \
       ASAN_OPTIONS='detect_leaks=1 detect_invalid_pointer_pairs=1 detect_stack_use_after_return=1' \
-      cargo test --lib --no-default-features --features="$ASAN_FEATURES" -Zbuild-std --target x86_64-unknown-linux-gnu
+      cargo test --lib --no-default-features --features="${ASAN_FEATURES:-}" -Zbuild-std --target x86_64-unknown-linux-gnu
     # There is currently a bug in the MemorySanitizer (MSAN) - disable the job for now.
     #
     # cargo clean
     # CC='clang -fsanitize=memory -fno-omit-frame-pointer'                                         \
     #   RUSTFLAGS='-Zsanitizer=memory -Zsanitizer-memory-track-origins -Cforce-frame-pointers=yes'   \
-    #   cargo test --lib --no-default-features --features="$ASAN_FEATURES" -Zbuild-std --target x86_64-unknown-linux-gnu
-}
-
-# Bench only works with a non-stable toolchain (nightly, beta).
-do_bench() {
-    RUSTFLAGS='--cfg=bench' cargo bench
-}
-
-# This is only relevant for hashes.
-do_schemars() {
-    cd "extended_tests/schemars" > /dev/null
-    cargo test
+    #   cargo test --lib --no-default-features --features="${ASAN_FEATURES:-}" -Zbuild-std --target x86_64-unknown-linux-gnu
 }
 
 # Check all the commands we use are present in the current environment.
 check_required_commands() {
     need_cmd cargo
     need_cmd rustc
-    need_cmd jq
```

Comment:
In d457ed8d54d72fdb7e09b69d38527f864f022a75:

Why is this removed?

Reply:
~`jq`  is not used anywhere.~ oooh its used in `test_vars.sh` - woops.

Reply:
\me chuckles - learned my grepping lesson from yesterday - win!